### DISCUSSION TITLE
[Solved] Color of cookie-logo?

### DISCUSSION THREAD

**Discussion by mave-git** [GetPublii/Publii#1445] [Help]
Hi,
I looked around the options in the gui, also inside css-files without success: is there an option changing the color of the (GPDR)-Cookie color? (Maybe only in some themes?)
Regards!

**Comment by bobmitro** [comment]
Yeap, it is. Just look at the docs https://getpublii.com/docs/gdpr-cookie-banner-configuration.html

**Comment by mave-git** [comment]
Ah - thanks! :-)
